Cargo.toml

```diff
@@ -66,7 +66,7 @@ terminal = { path = "crates/terminal" }
 
 subprocess = "0.2.9"
 tempfile = "3.8.0"
-tokio = { version = "1.23", features = ["full"] }
+tokio = { version = "1.40", features = ["full"] }
```

Comment:
This was necessary to make `fake-opentelemetry-collector` happy which relies on an up to date version of `tokio`.